amundsen_application/static/js/components/SearchPage/SearchBar/index.tsx

```diff
@@ -58,9 +58,13 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
   };
 
   handleValueSubmit = (event: React.FormEvent<HTMLFormElement>) : void => {
+    const searchTerm = this.state.searchTerm.trim();
     event.preventDefault();
     if (this.isFormValid()) {
```

Comment:
As a cleanup here, since we are now grabbing the `searchTerm` in this method and trimming it, we let `isFormValid` take that string, and perhaps at this point a better name is `isSearchTermValid`.

Reply:
:+1: cleaned this up